client/apps/edit/components/content/sections/image_collection/components/image_set.jsx

```diff
@@ -0,0 +1,35 @@
+import PropTypes from 'prop-types'
+import React from 'react'
+import {
+  ImageSetPreview,
+  ImageSetPreviewClassic
+} from '@artsy/reaction-force/dist/Components/Publishing'
+
+export const ImageCollectionImageSet = (props) => {
+  const { articleLayout, section } = props
+  const images = section.get('images') || []
+  let itemProps = {}
+
+  if (articleLayout === 'classic') {
+    itemProps = { images }
+
+    return (
+      <ImageSetPreviewClassic {...itemProps} />
+    )
+  } else {
+    itemProps.section = {
+      images,
+      layout: section.get('layout'),
+      title: section.get('title')
+    }
+
+    return (
+      <ImageSetPreview {...itemProps} />
+    )
+  }
+}
+
+ImageCollectionImageSet.propTypes = {
+  articleLayout: PropTypes.string.isRequired,
+  section: PropTypes.object.isRequired
+}
```

Comment:
Hmm it seems a little counter-intuitive to me that this exists in `/sections/image_collection` and has a name like `ImageCollectionImageSet` when those are two section types.

I feel like this file should belong in something like `sections/image_set`? I think what's happening is that the UI's are close enough that the `image_collection` and `image_set` _types_ can be co-located. If that's the case, it seems like a more general purpose name might be less confusing to peeps coming into the project, like `sections/images`.

Reply:
Yes i think sections/images is a good idea -- will update the naming